addons/account/models/account_move.py

```diff
@@ -1065,6 +1064,8 @@ def _compute_bank_partner_id(self):
                 move.bank_partner_id = move.commercial_partner_id
             else:
                 move.bank_partner_id = move.company_id.partner_id
+            filtred_bank_partner_id = move.bank_partner_id.filtered(lambda bank: bank.company_id.id in (False, move.company_id.id))
```

Comment:
There is a problem here as I am not usre invoice_partner_bank_id is a computed field and you do not depend on the company_id in the api.depends.  @abla001

Reply:
Indeed, because these are 2 stored computed fields, you need to compute them in two different functions